ott/core/sinkhorn_lr.py

```diff
@@ -383,6 +380,7 @@ def __call__(
         init_r = ot_sink_y.matrix
     else:
       raise NotImplementedError(self.init_type)
+    self.is_entropic = (self.epsilon != 0.0)
```

Comment:
I had in mind
```python
@property
def is_entropy(self) -> bool:
  return self.epsilon != 0.0
```
since it's read-only + we can write docs for this